src/main/java/com/snowflake/kafka/connector/internal/streaming/telemetry/SnowflakeTelemetryChannelCreation.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023 Snowflake Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.snowflake.kafka.connector.internal.streaming.telemetry;
+
+import static com.snowflake.kafka.connector.internal.telemetry.TelemetryConstants.CHANNEL_NAME;
+import static com.snowflake.kafka.connector.internal.telemetry.TelemetryConstants.IS_REUSE_TABLE;
+import static com.snowflake.kafka.connector.internal.telemetry.TelemetryConstants.START_TIME;
+import static com.snowflake.kafka.connector.internal.telemetry.TelemetryConstants.TABLE_NAME;
+
+import com.snowflake.kafka.connector.internal.telemetry.SnowflakeTelemetryBasicInfo;
+import com.snowflake.kafka.connector.internal.telemetry.SnowflakeTelemetryService;
+import net.snowflake.client.jdbc.internal.fasterxml.jackson.databind.node.ObjectNode;
+
+/**
+ * This object is sent only once when a channel starts. No concurrent modification is made on this
+ * object, thus no lock is required.
+ */
+public class SnowflakeTelemetryChannelCreation extends SnowflakeTelemetryBasicInfo {
+  private final long startTime; // start time of the channel
+  private final String channelName;
+  private boolean isReuseTable = false; // is the channel reusing existing table
+
+  public SnowflakeTelemetryChannelCreation(final String tableName, String channelName) {
+    super(tableName, SnowflakeTelemetryService.TelemetryType.KAFKA_CHANNEL_START);
+    this.channelName = channelName;
+    this.startTime = System.currentTimeMillis();
+  }
+
+  @Override
+  public void dumpTo(ObjectNode msg) {
+    msg.put(TABLE_NAME, this.tableName);
+    msg.put(CHANNEL_NAME, this.channelName);
```

Comment:
should we add `CONNECTOR_NAME` too? until we get connector name in our channel name? WDYT?

Reply:
Good point, added connector name to telemetry. Didn't add to metrics because it is surfaced in metric registry already